frontend/src/components/InstructorTable.tsx

```diff
@@ -0,0 +1,45 @@
+import {Instructor} from "../models/Instructor.ts";
+import {InstructorDto} from "../models/InstructorDto.ts";
+import {Grid, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Typography} from "@mui/material";
+import InstructorRow from "./InstructorRow.tsx";
+
+type InstructorTableProps= {
+    instructors: Instructor[];
+    deleteInstructor: (id: string) => void;
+    updateInstructor: (id: string, instructor: InstructorDto) => void;
+    loadingInstructor: boolean;
+}
+export default function InstructorTable( props: InstructorTableProps) {
+    const instructor: Instructor[]=props.instructors;
+    const instructorRows=instructor.map((instructor) => <InstructorRow key={instructor.id} loadingInstructor={props.loadingInstructor}
+                                                                    updateInstructor={props.updateInstructor}
+                                                                    deleteInstructor={props.deleteInstructor}
+                                                                    instructor={instructor}/>)
+    return (
+        <TableContainer component={Paper}>
+            <Table sx={{minWidth: 650}} aria-label="simple table">
+                <TableHead>
+                    <TableRow>
+                        <TableCell style={{ width: '10%' }} align={"center"} component={"th"}><strong>Vorname</strong></TableCell>
```

Comment:
I would recommend not using inline styling but rather styling in a CSS file or styled component